.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,4 +1,6 @@
+blank_issues_enabled: false
+
 contact_links:
-- name: "Crash bug"
-  url: https://istio.io/about/security-vulnerabilities/
-  about: "Please file any bug causing a crash to istio-security-vulnerability-reports@googlegroups.com."
+- name: "Issues"
+  url: https://issues.redhat.com/browse/MAISTRA
```

Comment:
can you replace this with `OSSM`? We migrated away from `MAISTRA`

Reply:
Fixed